File: codewars/go/7kyu/printer_errors.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	s := "aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbmmmmmmmmmmmmmmmmmmmxyz"
	print(PrinterError(s))
}

func PrinterError(s string) string {
	allowedLetters := "abcdefghijklm"
	errorsCounts := 0
	for _, letter := range s {
		if !strings.Contains(allowedLetters, string(letter)) {
			errorsCounts++
		}
	}
	return fmt.Sprintf("%v/%v", errorsCounts, len([]rune(s)))
}

File: codewars/python/7kyu/printer_errors.py
def printer_error(s: str) -> str:
    allowedLetters = 'abcdefghijklm'
    errorsCount = 0
    for letter in s:
        if letter not in allowedLetters:
            errorsCount += 1
    return f'{errorsCount}/{len(s)}'


if __name__ == '__main__':
    s = "aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbmmmmmmmmmmmmmmmmmmmxyz"
    print(printer_error(s))
